projects/cashmere/src/lib/radio-button/radio-button.md

```diff
@@ -1,28 +1,32 @@
-##### Radio Text Formatting
+##### Button Toggle Text Formatting
 
 For both the label on radio buttons and their contents, please use the following formatting:
 
 Capitalize first word only, except in cases where the selection is an actual program name or proper name (e.g. Patient non-adherence or challenges in care management; Duplicate patient; Algorithm incorrect; Comprehensive HIV AIDS Management Program (CHAMP), Grace Program, LSU Diabetes, Salt Lake City, etc.).
 
 &nbsp;
 
-##### Radio Group Labels
+##### Button Toggle Group Labels
 
 Use a colon at the end of radio group labels (e.g. Email address:). A colon implies a direct connection between the label text and a particular control or set of controls. The only situation where a colon should not be used is when the label and control work together to form a single sentence.
 
 &nbsp;
 
+#### Button Toggle Icons
+
+&nbsp;
```

Comment:
Nothing going in here?

Reply:
Good catch @jonest - looks like this was accidental.  I'll get roll back a few of the changes to this markdown file.